src/utils/client.js

```diff
@@ -35,6 +36,16 @@ async function connectToServer(url) {
     }
   };
 
+  if (secret) {
```

Comment:
So I get it because Cerner doesn't support JWT client assertion we need to add this but the SMART Backend Authorization spec says it SHALL use JWT and not client secret. I know its easier for us to adapt than make Cerner be compliant but it still rubs me the wrong way. I think at a minimum letting them know and including a comment here saying this is out of spec and only to support Cerner would help

Reply:
Cerner doesn't support subscriptions, right? So is this even needed here or only in the subscriptions-proxy? Since the interface between the proxy and the sandbox is out of scope of the spec I'm totally fine with having that use client secret

Reply:
That's a good point. This is a reference implementation of a specification. Cerner doesn't claim to support the MedMorph spec or the SMART backend auth spec so adding code here to support them is strange. Cerner and Epic don't support either the Subscription resource, nor any approach to build a subscription capability on top of what they do support, so I'm not sure if it's worth using the subscription proxy at all. (They require all resource queries to be by ID, so for example you can get an Encounter by ID, or all Encounters by a Patient ID, but not all Encounters.) I guess we could use it as an "auth shim"...